src/util/profile.c

```diff
@@ -5,21 +5,21 @@
 #endif
 
 struct ProfileData {
-    u64 lastReportStart;
-    u64 timeAccumulation[MAX_PROFILE_BINS];
+    uint64_t lastReportStart;
+    uint64_t timeAccumulation[MAX_PROFILE_BINS];
 };
 
 struct ProfileData gProfileData;
 
-void profileEnd(u64 startTime, int bin) {
-    gProfileData.timeAccumulation[bin] += OS_CYCLES_TO_USEC(osGetTime() - startTime);
+void profileEnd(Time startTime, int bin) {
+    gProfileData.timeAccumulation[bin] += timeMicroseconds(timeGetTime() - startTime);
 }
 
 void profileReport() {
 #ifdef PORTAL64_WITH_DEBUGGER
-    OSTime reportStartTime = osGetTime();
+    uint64_t reportStartTime = timeMicroseconds(timeGetTime());
```

Comment:
There is some loss of precision by not storing it in cycles, but given the use case this should be fine.